docs/01-overview/main-areas/service-mesh/smsh-04-istio-sidecars-in-kyma.md

```diff
@@ -0,0 +1,37 @@
+
+Kiali is another tool that comes as [separate Kyma component](../../../05-technical-reference/00-architecture/obsv-04-architecture-kiali.md) and Kyma configures Istio to export metrics necessary to support Kiali features that facilitates managing, visualizing and troubleshooting your mesh.
+
+Moreover, Kyma provides [Istio specific dashboards](https://istio.io/latest/docs/ops/integrations/grafana/#configuration) for [Grafana (monitoring component)](../../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md) and together with metrics exposed by the sidecar proxy provides better visibility into workloads and mesh control plane performance.
+
+The sidecar proxy enables all the advanced observability, that would not be possible without instrumentation code within your application that could complicate your domain code.
+
+## Traffic management
+
+[Traffic management](https://istio.io/latest/docs/concepts/traffic-management/) is an important feature of service mesh and having the sidecar injected into every workload makes this feature available without additional configuration.
+
+[Traffic Shifting](https://istio.io/latest/docs/tasks/traffic-management/traffic-shifting/) and [Request Routing](https://istio.io/latest/docs/tasks/traffic-management/request-routing/) allows developers to release their software using techniques like canary releases and A/B testing to make release process faster and more reliable.
+
+[Mirroring](https://istio.io/latest/docs/tasks/traffic-management/mirroring/) and [Fault Injection](https://istio.io/latest/docs/tasks/traffic-management/fault-injection/) can be used for testing and audit purposes to improve resiliency of your applications.
+
+### Resiliency
+
+Application resiliency is an important topic within traffic management functionalities. Traditionally resiliency features like timeouts, retries and circuit breakers were implemented by application libraries, however with service mesh you can delegate that tasks to the mesh and the same configuration options will work regardless of the programming language of your application. You can read more about it in [Network resilience and testing](https://istio.io/latest/docs/concepts/traffic-management/#network-resilience-and-testing)
+
+## Sidecar proxy behavior during Kyma upgrade.
+
+When upgrading Kyma we want to have full compatibility of existing workloads with the upgraded version of Istio. To achieve that we try to perform `rollout restart` of the workloads whenever possible because this ensures that newest version of sidecar proxy will be injected into the pods. There are exception when it's not possible to restart workloads and the detailed behaviour is described in [troubleshooting guide](https://kyma-project.io/docs/kyma/latest/04-operation-guides/troubleshooting/apix-09-upgrade-sidecar-proxy#cause) 
```

Comment:
```suggestion
When upgrading Kyma we want to have full compatibility of existing workloads with the upgraded version of Istio. To achieve that we try to perform `rollout restart` of the workloads whenever possible because this ensures that newest version of sidecar proxy will be injected into the pods. There are exceptions when it's not possible to restart workloads and this behaviour is described in [troubleshooting guide](https://kyma-project.io/docs/kyma/latest/04-operation-guides/troubleshooting/apix-09-upgrade-sidecar-proxy#cause) 
```